lib/presentation/dashboard_screen/widget/custom_app_bar.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:sufi_ishq/core/app_export.dart';
+import 'package:sufi_ishq/core/utils/constant.dart';
+import 'package:sufi_ishq/presentation/dashboard_screen/controller/dashboard_controller.dart';
+import 'package:sufi_ishq/theme/color_initializer.dart';
+
+class CustomAppBar extends StatelessWidget {
```

Comment:
change class name currently it's not relevant

Reply:
Done